spec/system/detailed_guide_spec.rb

```diff
@@ -0,0 +1,138 @@
+
+    it "displays historically political detailed guide page" do
+      within(".govuk-notification-banner__content") do
+        expect(page).to have_content("This was published under the 2010 to 2015 Conservative and Liberal Democrat coalition government")
+      end
+    end
+  end
+
+  context "when visiting detailed guide that applies to a set of nations only" do
+    let(:content_store_response) { GovukSchemas::Example.find("detailed_guide", example_name: "national_applicability_detailed_guide") }
+
+    it "displays detailed guide page that applies to a set of nations" do
+      expect(page).to have_devolved_nations_component("Applies to England")
+    end
+  end
+
+  context "when visiting detailed guide that applies to a set of nations with alternative URLs" do
+    let(:content_store_response) { GovukSchemas::Example.find("detailed_guide", example_name: "national_applicability_alternative_url_detailed_guide") }
+
+    it "displays detailed guide page that applies to a set of nations having alternative URLs" do
+      expect(page).to have_devolved_nations_component("Applies to England, Scotland and Wales", [
+        {
+          text: "Guidance for Northern Ireland",
+          alternative_url: "http://www.dardni.gov.uk/news-dard-pa022-a-13-new-procedure-for",
+        },
+      ])
+    end
+  end
+
+  context "when visiting a translated detailed guide page" do
+    let(:content_store_response) { GovukSchemas::Example.find("detailed_guide", example_name: "translated_detailed_guide") }
+
+    it "displays available translations for detailed guide page" do
+      within(".gem-c-translation-nav") do
+        expect(page).to have_css(".gem-c-translation-nav__list-item", text: "English")
+        expect(page).to have_link("Cymraeg", href: "/guidance/prepare-a-charity-annual-return.cy")
+      end
+    end
+  end
+
+  context "when visiting a detailed guide page with logo" do
+    let(:content_store_response) { GovukSchemas::Example.find("detailed_guide", example_name: "england-2014-to-2020-european-structural-and-investment-funds") }
+
+    it "conditionally renders a logo" do
+      expect(page).to have_css(".metadata-logo[alt='European structural investment funds']")
+    end
+  end
+
+  context "when checking for single page notification button" do
+    it "renders with the single page notification button on English language pages" do
+      expect(page).to have_css(".gem-c-single-page-notification-button")
+
+      buttons = page.all("button[data-ga4-link]")
+      expected_tracking_top = single_page_notification_button_ga4_tracking(1, "Top")
+      actual_tracking_top = JSON.parse(buttons.first["data-ga4-link"])
+      expect(actual_tracking_top).to eq(expected_tracking_top)
+
+      expected_tracking_bottom = single_page_notification_button_ga4_tracking(2, "Footer")
+      actual_tracking_bottom = JSON.parse(buttons.last["data-ga4-link"])
+      expect(actual_tracking_bottom).to eq(expected_tracking_bottom)
```

Comment:
non-blocking nitpick - except where it's the first line in a block or following another expect line, could we always have a space before each expect?